mdaviz/mda_folder_table_model.py

```diff
@@ -11,30 +11,17 @@
 from PyQt5 import QtCore
 from . import utils
 
+HEADERS = "Prefix", "Scan #", "Points", "Dim", "Positioner", "Date", "Size"
+
 
 class MDAFolderTableModel(QtCore.QAbstractTableModel):
     def __init__(self, data, parent):
-        self.parent = parent
-
-        self.actions_library = {
-            "Prefix": lambda file: file.rsplit("_", 1)[0],
-            "Scan #": lambda file: int(file.rsplit("_", 1)[1].split(".")[0]),
-            "Points": lambda file: self.get_file_pts(file),
-            "Dim": lambda file: self.get_file_dim(file),
-            "Positioner": lambda file: self.get_file_pos(file),
-            "Date": lambda file: self.get_file_date(file),
-            "Size": lambda file: self.get_file_size(file),
-        }
-
-        self.columnLabels = list(self.actions_library.keys())
-
-        self.setAscending(True)
-        self._folderCount = 0
-
+        # parent = <mdaviz.mda_folder.MDA_MVC object at 0x1101e7520>
```

Comment:
Consider converting this comment into a docstring that describes the `parent` parameter.  Here's an example:

```
    """
    Create the model and connect with its parent.

    PARAMETERS

    parent object:
        Instance of mdaviz.mda_folder.MDAMVC
    """
```

Reply:
And apply to other similar instances.  Alternative is to rename the parameter from `parent` to `mvamvc_parent`.

Reply:
That comment was a note to myself during debugging. I was trying to keep track of the parent-child relationships. I can update the docstrings. I did rename `self.parent` to `self.mda_mvc`.

Reply:
I updated all the classes with docstrings https://github.com/BCDA-APS/mdaviz/pull/91/commits/27b27ba6ad5c436157386b18be78cd1d215e6780